src/styles/utils/normalize.scss

```diff
@@ -0,0 +1,8 @@
+body {
+  display: flex;
+  justify-content: center;
+  align-items: center;
```

Comment:
Don't set such properties in normalize.scss

Reply:
Okey